pkg/ebpf/telemetry/modifier.go

```diff
@@ -26,13 +26,33 @@ const (
 )
 
 // ErrorsTelemetryModifier is a modifier that sets up the manager to handle eBPF telemetry.
-type ErrorsTelemetryModifier struct{}
+type ErrorsTelemetryModifier struct {
+	trackedMaps []names.MapName
+}
 
 // String returns the name of the modifier.
 func (t *ErrorsTelemetryModifier) String() string {
 	return "ErrorsTelemetryModifier"
 }
 
+// getMapNames returns the names of the maps in the manager.
+func getMapNames(m *manager.Manager, opts *manager.Options) []names.MapName {
+	var mapNames []names.MapName
+	for _, mp := range m.Maps {
+		mapNames = append(mapNames, names.NewMapNameFromManagerMap(mp))
+	}
+
+	for mpName, _ := range opts.MapEditors {
+		spec, _, err := m.GetMapSpec(mpName)
+		if err != nil {
+			continue
+		}
+		mapNames = append(mapNames, names.NewMapNameFromMapSpec(spec))
+	}
```

Comment:
explain why this is needed, in addition to iterating over the manager's maps

Reply:
No longer applicable since I am only use the MapSpecs list not https://github.com/DataDog/datadog-agent/pull/32650/commits/776db5cc1ac3f2cc6a657dfbe8c65bd7a3bf95e1